compiler/pota-quantization-value-test/requires.cmake

```diff
@@ -2,5 +2,5 @@ require("record-minmax")
 require("circle-quantizer")
 require("circle-tensordump")
 require("common-artifacts")
-require("mio-circle04")
+require("mio-circle05")
```

Comment:
I think we can remove depends to `mico-circleXX` for this module if there is no direct dependency from any source.